.NET/Microsoft.Recognizers.Text.DateTime/Parsers/FullDateTimeParser.cs

```diff
@@ -417,10 +417,9 @@ public DateTimeParseResult Parse(ExtractResult er, DateObject referenceTime)
                 }
             }
 
-            if (!string.IsNullOrEmpty(comment) && comment.Equals(Constants.Comment_ContainFeb29th, StringComparison.Ordinal))
+            if (!string.IsNullOrEmpty(comment) && comment.Equals(Constants.Comment_DoubleTimex, StringComparison.Ordinal))

```

Comment:
Minor comment. But 
`comment.Equals(Constants.Comment_DoubleTimex, StringComparison.Ordinal)`
could be a method HasDoubleTimex or HasCompositeTimex. Just to improve code readability.